util/regexp-assemble/regexp-933160.data

```diff
@@ -1,3 +1,19 @@
+##! This is a data file used to generate a regular expression for a CRS rule. 
```

Comment:
Looks good. But maybe a prefix and suffix comment could be added instead of adding them after assembling the rule.

Reply:
Missed that one, thanks. Fixed.